modules/xfem/src/userobjects/MeshCut2DFractureUserObject.C

```diff
@@ -82,8 +105,8 @@ MeshCut2DFractureUserObject::initialize()
 void
 MeshCut2DFractureUserObject::findActiveBoundaryGrowth()
 {
-  // k1 is empty (but not a nullptr) on the very first time step because this UO is called before
-  // the InteractionIntegral or crackFrontStress vpp
+  // k_vpp's & stress_vpp are empty (but not a nullptr) on the very first time step because this UO
```

Comment:
```suggestion
  // The k*_vpp & stress_vpp are empty (but not a nullptr) on the very first time step because this UO
```

Reply:
This confused me because the possessive "'s" made me think it was referring to the "& stress_vpp" owned by the k_vpp.